lib/Core/Input/JsonInput.php

```diff
@@ -208,6 +212,26 @@ private function processFields(array $values)
         }
     }
 
+    private function processOrder(SearchCondition $condition, $array)
+    {
+        $order = $array['order'] ?? [];
```

Comment:
Switch to the same approach as in in StringInput, separate conditions builder for order?